### PR TITLE
Button

### PR DESCRIPTION
**Pull request by mehul-anand**
### Primary Button
- Default 
- Primary 
- Secondary 
- Error
- Warning 
- Success

### Outline Button
- Default 
- Primary 
- Secondary 
- Error
- Warning 
- Success

### CODE REVIEW

File: button.css [hunk 1]
```diff
@@ -0,0 +1,66 @@
+.button{
+    background-color: var(--default-color);
+    color: var(--off-white);
+    min-width: 100px;
```
Comment (nayyyhaa) [inline button.css:4]: Just a suggestion, you could use `rem` units for widths. Rest the code looks good to merge. 👍

**Review by nayyyhaa — APPROVED** [review]

File: button.css (hunk 1 above)
Comment (mehul-anand) [reply button.css:4]: Got it
Thanks a lot

**Review by Hamza-777 — APPROVED** [review]
Code looks clean and good.

**Review by ApurvaSawant11 — APPROVED** [review]
Looks good to me :)

**Pull request merged**